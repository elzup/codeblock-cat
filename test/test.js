const mdCat = require("../");
const util = require("util");
const spy = jest.spyOn(process.stdout, "write");

it("cat list.txt", () => {
	mdCat("test/list.txt");

	expect(spy.mock.calls[0][0]).toMatchSnapshot();
	expect(new util.TextDecoder().decode(spy.mock.calls[1][0])).toMatchSnapshot();
	expect(spy.mock.calls[2][0]).toMatchSnapshot();
});
