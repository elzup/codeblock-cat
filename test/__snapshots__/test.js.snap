// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cat list.txt 1`] = `"\`\`\`txt title=test/list.txt"`;

exports[`cat list.txt 2`] = `
"apple
banana
cat
"
`;

exports[`cat list.txt 3`] = `"\`\`\`"`;
